src/main/webapp/app/guided-tour/guided-tour.model.ts

```diff
@@ -24,4 +24,6 @@ export interface GuidedTour {
      * This should only be set if you are completely sure your tour is displaying correctly on all screen sizes otherwise a user can get stuck.
      */
     preventBackdropFromAdvancing?: boolean;
+    /** Defines if the UML model in the apollon editor should be resetted if the user restarts the tutorial */
+    resetUMLModel?: boolean;
```

Comment:
this is not good, the guided tour should not depend on a `resetUMLModel` boolean. 
A better approach would be to offer a `restart` or `reset` callback that can be executed with specific steps. We discussed this already and I think before adding such attributes, we should immediately handle this properly!

Reply:
@krusche I have created an issue for that and will implement it in a follow-up PR: https://github.com/ls1intum/Artemis/issues/1062